v2/pkg/protocols/headless/request.go

```diff
@@ -215,3 +226,16 @@ func (request *Request) executeFuzzingRule(input *contextargs.Context, payloads
 	}
 	return nil
 }
+
+// getLastNaviationURL returns last successfully navigated URL
+func (request *Request) getLastNaviationURLWithLog(reqLog map[string]string) string {
```

Comment:
`getLastNavigationURL`

Reply:
Copilot is good and bad at same time 🥲 . Lot of typosss!